HSTracker/UIs/DeckManager/DeckManager.swift

```diff
@@ -475,6 +477,28 @@ extension DeckManager: NSTableViewDelegate {
                 cell.image.image = NSImage(named: deck.playerClass.lowercaseString)
                 cell.color = ClassColor.color(deck.playerClass)
                 cell.selected = tableView.selectedRow == -1 || tableView.selectedRow == row
+                
+                switch sortCriteria {
+                case "creation date":
+                    let formatter = NSDateFormatter()
+                    formatter.dateStyle = .MediumStyle
+                    formatter.timeStyle = .NoStyle
+                    cell.detailTextLabel.stringValue =
+                        "\(formatter.stringFromDate(deck.creationDate!))"
+                case "win percentage":
+                    cell.detailTextLabel.stringValue = "\(deck.displayStats())"
+                case "wins":
+                    cell.detailTextLabel.stringValue = "\(deck.wins()) " +
+                        NSLocalizedString("wins", comment: "").lowercaseString
+                case "losses":
+                    cell.detailTextLabel.stringValue = "\(deck.losses()) " +
+                        NSLocalizedString("losses", comment: "").lowercaseString
+                case "games played":
+                    cell.detailTextLabel.stringValue = "\(deck.statistics.count) " +
+                        NSLocalizedString("games", comment: "").lowercaseString
+                default:
+                    cell.detailTextLabel.stringValue = ""

```

Comment:
I would remove the `win percentage` case and set it as `default`, this way, you always have something